Description: Change the target installation directory
 Presets will be modified to use the command line, after
 the dust settles.
Author: Julien Blache <jblache@apple.com>

Index: swiftlang-5.6.0/swift/utils/build-presets.ini
===================================================================
--- swiftlang-5.6.0.orig/swift/utils/build-presets.ini
+++ swiftlang-5.6.0/swift/utils/build-presets.ini
@@ -831,7 +831,7 @@ install-swiftpm
 install-swift-driver
 install-xctest
 install-libicu
-install-prefix=/usr
+install-prefix=/opt/swift/5.6.0
 install-libcxx
 install-sourcekit-lsp
 install-swiftdocc
